content/posts/sharing-data-via-torrent.md

```diff
@@ -44,31 +44,33 @@ To understand how this may impact a network, we first need to discuss how torren
+
+Finally, torrenting scales exceptionally well. If a large number of users try to download data from a central server at the same time, this can put immense load on a network, causing performance degradation. However, when downloading data via BitTorrent, each client is sharing bandwidth, reducing the load on any one server.
 
 Amazing! We have:
 
-1. Reduced Bandwidth Strain on Your Network,
-2. Faster Download Speeds,
-3. More Reliable Downloads,
-4. Decentralization and,
-5. Better Scalability
+1. High performance
+2. High reliability
+3. High availability
+4. High scalability
+5. Lower network load
 
 ## Why Would We Ever _Not_ Use the Torrent Protocol to Share Our Data?
 
-They say sharing is caring, and when it comes to torrenting, that couldn't be more true. If you are the only seed and there are no other peers, then the downloader is using only your bandwidth to retrieve the file. In this scenario, the stress on your network is essentially the same as a direct download because you're the sole source of the data. So, when it comes to torrenting, having multiple people seeding and/or as peers is essential to not only its functionality but to take advantage of all of the benefits it provides. For professionals and hobbyists to effectively share data via torrents, the data must be in demand. If you work in a highly specialized field with few potential downloaders or handle proprietary company data, maintaining a healthy number of seeders and peers can be a challenge. So, sharing your data via the torrent protocol might seem ideal in theory, but real-world results can vary.
+They say sharing is caring, and when it comes to torrenting, that couldn't be more true. If you are the only seeder and there are no other peers, then the downloader is using only your bandwidth to retrieve the file. This scenario is essentially the same as a centralized, direct download because you're the sole source of the data. That means having multiple seeders and peers are essential for BitTorrent to work effectively. For professionals and hobbyists to effectively share data via torrents, the data must be in demand. If you work in a highly specialized field with few potential downloaders or handle proprietary company data, maintaining a healthy number of seeders and peers can be a challenge. So, sharing your data via the BitTorrent protocol might seem ideal in theory, but real-world results can vary.
 
-When downloading chunks of data from different peers there is always a chance that one of those chunks may be faulty. Thankfully, the risk of this is minimal as the torrent protocol has built-in hash checks to verify each piece of the file, preventing most corruption issues.
+When downloading chunks of data from different peers there is always a chance that one of those chunks may be faulty. Thankfully, the risk of this is minimal as the BitTorrent protocol has built-in hash checks to verify each piece of the file, preventing most corruption issues.
```

Comment:
Again, BitTorrent is the protocol name

Reply:
Another reason for not using torrents that might be considered is it's reputation for the distribution of piracy, and that some organizations (especially university campuses) might block P2P traffic like BitTorrent under the (crappy) assumption that all traffic must be malicious traffic.